### ISSUE TITLE
Find out how people use lobbying data

### ISSUE THREAD

**Issue by rofreg** [codeisland/ri-lobbying#2]
We're not entirely sure what we're building yet, other than a basic app to view lobbying data. We need other people to help shape our intuitions – people who have more experience working with lobbying data than we do.

We should talk to people who already look at lobbying data, and ask them:
- What do you currently do with lobbying data?
- What do you *wish* you could do with lobbying data?
- What obstacles cost you the most time/effort?

Types of people to talk to include:
- Journalists
- Activists
- Lobbyists themselves
- Others?

Anyone have any leads here? I know some people at Common Cause who might have good thoughts about this.

**Comment by paulopperman** [comment]
I can pass this along to some folks who might be able to help.

**Comment by rofreg** [comment]
Nicole L (via [Slack](https://codeisland.slack.com/archives/C02P8F2K2/p1512489405000290)) was nice enough to contact a few active users of the current Lobby Tracker and ask if they'd be willing to talk to us. I've reached out to two of them already, hoping to schedule chats in the next ~2 weeks so that I can pick their brains about the existing Lobby Tracker. If you'd like to join those conversations, please let me know – I'd love the company!

- [x] John Marion of Common Cause (john_marion@commoncauseri.org)
  - Meeting on Friday, Dec. 15 at 9:00am
- [x] Ted Nesi (tnesi@wpri.com)
  - Meeting on Tuesday, <strike>Dec. 12</strike> Dec. 19 at 10:00am

Nicole also identified a few more people that I have not contacted yet. If anyone would be interested in meeting with these people and gathering notes and product ideas, I think that could be really useful.

- [ ] Bill Rappleye of Channel 10 (WCRappleye@sbgtv.com)
- [ ] Ian Donnis (idonnis@ripr.org)
- [ ] Patrick Anderson (panderson@providencejournal.com)
  - Patrick was hesitant based on time commitment
- [ ] Kathy Gregg (kgregg@providencejournal.com)
  - "Probably the heaviest user of the lobbying data", according to Nicole. She's pretty busy, so she may have limited time to discuss.
- [ ] Chris Barnett (cbarnett@rifoundation.org)

**Comment by rofreg** [comment]
Some brief notes from the meeting that @goldenmeanie and I had with John Marion:

### What are the most valuable updates that we could make to the existing lobbying tracker?
  - **Combine the data from the old database and the new database**
    - Right now, users have to search through both, and it's a pain
  - **Show the history of an Individual over time (e.g. Jane Doe)**
    - "Was Jane Doe lobbying in 2015? Who for?"
    - The current system lets you look up records for individual years, but it doesn't summarize one person's activity over many years
  - **Show the history of an Entity over time (e.g. Verizon)**
    - "Who has lobbied for Verizon over the past decade?"
    - "How much has Verizon spent on lobbying since 2010?"
    - Again, the current system is poor for this purpose

### Additional things that would be neat (but may be more difficult/impossible)
- The old database has a lot of numeric data in non-numeric formats. Converting it would be great, but it might be EXTREMELY difficult, as the fields are strings in no particular format (e.g. `$10000`, `$3k/month`, `$1,200 per quarter`, etc etc).
- Connecting this DB to campaign contribution data would be fantastic, but very hard
- Connecting to Board of Elections campaign spending data would also be fantastic + hard
- The new database has data about which specific bills people were lobbying for, which could be interesting...
  - ...but the field is optional, so there may not be many people filling it in

### Other important notes
- Our lobbying tracker will not be useful if the data is not current. Ongoing updates will be critical.
  - To what extent should we be trying to build an independent, 3rd-party tracker, vs. building something that we expect the Secretary of State's office to take in-house and run?
- Existing users are not looking for API access (they are generally not that technical), but they may(?) appreciate some other form of export. John personally has not had reason to use export features in the past
- Code Island may be able to work on multiple workstreams in parallel
  - Workstream 1: Import data from the old database into the new database
  - Workstream 2: Import new monthly data from the state into our independent lobbying tracker
  - Workstream 3: Build a web interface that makes the data easier to consume, as described above
  - Workstream 4: Build an API for other people to build off of

**Comment by amygriffis** [comment]
Notes from the meeting with Ted Nesi:

### How do you use lobbying data?

- looking for mentions, crumbs to follow a trail
  - this makes search much more useful than a navigation-based interface
  - often use google site search to look for info
- recent data is the most valuable
- need high-level summaries
  - if something comes up, then want the details
  - don't mind digging through archives or other data formats at this point in the process
- trying to weave together the story of the relationships
  - people connections matter more than the money numbers

### What obstacles cost you the most time/effort?

- biggest problem is the split db
  - initially was unclear that 2017 data exists
  - google search points to the old db, which has an empty entry for 2017 and no pointer to the new db
  - need to see relationships from the current year going back, right now have to piece this together
- spend a lot of time copying numbers from reports and tallying them in google sheets
  - having the site do tallies would reduce time and human error
- hard to see discrepancies between send and receive amounts, due to having to navigate through each side
  - would like to see who's not reporting properly (also mentioned by John Marion)
  - the site could make this much more visible than it currently does

### What do you *wish* you could do with lobbying data?

- tie lobbying data to bills
  - would like to enter a bill#, see who's lobbying
    - currently an optional field, so no one fills it in
  - or what bills did [lobbyist name] testify on this year?
    - the list of people who testify on a given bill is recorded, but in another part of govt and completely separate from this data
- other nice-to-haves
  - data export
  - comparison of lobby org spending
  - mining other data systems

**Comment by amygriffis** [comment]
### Ted's Interface Comments

_In general: don't need to throw everything out, but it needs to be more
intuitive._

#### Landing page
https://apps.sos.ri.gov/lobbytracker/profiles

- it's not clear when new data is available, would like to see a listing of
  the latest reports
- having to choose a term (entities, lobbying firms, lobbyists) is a tough
  starting point, don't always know which one to choose
- global search would be helpful, current search just filters the selected
  term
- would be nice to see summary info, eg. highest paid _(probably not
  possible if govt hosted)_
- financial summary could be interesting for entities, maybe lobbyist
  firms, actual or estimated for 2017

#### Subjects tab
https://apps.sos.ri.gov/lobbytracker/profiles/subjects

- needing to manually expand all the entries is cumbersome

#### Lobbyist details page
https://apps.sos.ri.gov/lobbytracker/profiles/view/642

- type sizes make this page confusing, biggest headline should be most
  important
- most important information is

1. who
2. relationships
3. submitted reports

- confusing terms: what is sender, receiver, $?
- zeros under $ are confusing, N/A might be more appropriate?
    - example: Lifespan -> JPM & Assoc -> Joan P. Milas
- do the submitted reports have more info, or are they just more dry and
  bureaucratic? do i need to open them? what else is there that i'm not
  seeing?

#### Open Govt Interactive & Access Report
http://sos.ri.gov/divisions/open-government/Transparency/open-government-interactive

- contains helpful summary data, but completely buried and not easily
  findable
- including links to related sites like this would be very helpful

#### Examples

RI Campaign Finance
http://www.ricampaignfinance.com/RIPublic/HomePage.aspx

- an old site, but good example of showing the latest filings

#### OpenSecrets.org

- check out their lobbying section, https://www.opensecrets.org/lobby/
- summaries are good (cycle time, events, tops of various categories)
- who's lobbying, who's spending the most
- ted uses the legislative campaign spending section on this site
